sourmash/lca/lca_utils.py

```diff
@@ -25,6 +27,21 @@
 LineagePair = namedtuple('LineagePair', ['rank', 'name'])
 
 
+def cached_property(fun):
```

Comment:
this is a missed opportunity to make the code 3.8+ only =P
https://docs.python.org/3/library/functools.html#functools.cached_property

Reply:
thank you for your comment.